// common.proto

// this file contains the datatypes definitons commonly used by these grpc servers 
// to send data between them

syntax = "proto3";

package common;

message TableData {
    string table_name = 1;
    bytes data = 2;
}

message EmptyRequest {}

message LoadResponse {
    string message = 1;
}
